dapps/react-dapp-v2/src/contexts/JsonRpcContext.tsx

```diff
@@ -1,4 +1,4 @@
-import { BigNumber, utils } from "ethers";
+import { BigNumber, getDefaultProvider, utils } from "ethers";
```

Comment:
Unused import:

```suggestion
import { BigNumber, utils } from "ethers";
```

Reply:
> Closes #117
> 
> Thank you for moving to fix this already @basheerAudteye pray
> 
> Do you think it's worth also applying this to the `eth_sign` flow directly below?

in order to make non-custodial wallets fully functional with walletConnect V2 we need to apply the same logic to eth_signTypedData (eip-712) and eth_sign also